tests/shakedown/shakedown/dcos/master.py

```diff
@@ -174,3 +178,23 @@ def disconnected_master(incoming=True, outgoing=True):
     finally:
         # return config to previous state
         reconnect_master()
+
+
+@lru_cache()
+def dcos_masters_public_ips():
+    """
+    retrieves public ips of all masters
+
+    :return: public ips of all masters
+    """
+    @retrying.retry(
+        wait_fixed=1000,
+        stop_max_attempt_number=240,  # waiting 20 minutes for exhibitor start-up
```

Comment:
By my calculations, this is 4 minutes; am misreading something?

Reply:
Nope. I think the comment next to it is outdated or a typo. We should not wait 20 minutes.